errbot/errbot.py

```diff
@@ -140,7 +140,7 @@ def get_config(config_path):
     return config
 
 
-if __name__ == "__main__":
```

Comment:
@gbin I was under the impression that in the dev environment you will always do `python setup.py develop` and therefore this will not be necessary but I will gladly add such option.